packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -671,14 +671,18 @@ class ListTile extends StatelessWidget {
   /// if it's not null and to [Colors.transparent] if it's null.
   final Color? selectedTileColor;
 
+  /// {@template flutter.material.ListTile.enableFeedback}
   /// Whether detected gestures should provide acoustic and/or haptic feedback.
   ///
   /// For example, on Android a tap will produce a clicking sound and a
   /// long-press will produce a short vibration, when feedback is enabled.
+  /// 
+  /// When null, the default value is true.
   ///
   /// See also:
   ///
   ///  * [Feedback] for providing platform-specific feedback to certain actions.
+  /// {@endtemplate}
```

Comment:
We tend to not include the "See also" section in templates, since if you use the template somewhere, you might want to add different references or combine it with other test, and you wouldn't be able to do that easily if the "See also" is embedded in the template.

Can you move this `{endtemplate}` up to before the "See also", and add the see also to the places that use the template?

Reply:
Sure, I've pushed the code.